pyproject.toml

```diff
@@ -105,7 +105,8 @@ pyside = [
     "napari[pyside2]"
 ]
 pyqt5 = [
-    "PyQt5>=5.15.8"
+    "PyQt5>=5.15.8",
+    "pyqt5-qt5<=5.15.2; sys_platform == 'Windows'",  # 5.15.2 is latest to build on Windows, required for `uv add` compatability as of uv 0.6.9
```

Comment:
I think that you would also need to specify that `pyqt5-qt5` would be need to be greater than 5.15.8 for !windows